templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content%}
+<h1>Delete manufacturer</h1>
+<p>Are you sure wont to delete the manufacturer: {{ manufacturer.name }}</p>
```

Comment:
There is a typo in the confirmation message: 'wont' should be corrected to 'want'. The line should read: `<p>Are you sure you want to delete the manufacturer: {{ manufacturer.name }}</p>`.